Config/Media.json

```diff
@@ -1,4 +1,88 @@
-[{
+[
+{
+    "Id":  "2016_x64_Standard_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Standard 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERSTANDARD",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
+    "Checksum":  "18A4F00A675B0338F3C7C93C4F131BEB",
+    "CustomData":  {
+        "WindowsOptionalFeature":  ["NetFx3"]
+    },
+    "Hotfixes":  []
+},
+{
+    "Id":  "2016_x64_Standard_Core_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Standard Core 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERSTANDARDCORE",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
+    "Checksum":  "18A4F00A675B0338F3C7C93C4F131BEB",
+    "CustomData":  {
+        "WindowsOptionalFeature":  ["NetFx3"]
+    },
+    "Hotfixes":  []
+},
+{
+    "Id":  "2016_x64_Datacenter_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Datacenter 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERDATACENTER",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
+    "Checksum":  "18A4F00A675B0338F3C7C93C4F131BEB",
+    "CustomData":  {
+        "WindowsOptionalFeature":  ["NetFx3"]
+    },
+    "Hotfixes":  []
+},
+{
+    "Id":  "2016_x64_Datacenter_Core_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Datacenter Core 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERDATACENTERCORE",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
+    "Checksum":  "18A4F00A675B0338F3C7C93C4F131BEB",
+    "CustomData":  {
+        "WindowsOptionalFeature":  ["NetFx3"]
+    },
+    "Hotfixes":  []
+},
+{
+    "Id":  "2016_x64_NANO_EN_Eval",
```

Comment:
There are actually two Nano images:
```powershell
C:\..\Dropbox\PowerShell\> Get-WindowsImage -ImagePath D:\Nano\NanoServer.wim

ImageIndex       : 1
ImageName        : Windows Server 2016 SERVERSTANDARDNANO
ImageDescription :
ImageSize        : 800,233,355 bytes

ImageIndex       : 2
ImageName        : Windows Server 2016 SERVERDATACENTERNANO
ImageDescription :
ImageSize        : 811,183,747 bytes
```
Rename this one to `2016_x64_Standard_Nano_EN_Eval`